package.json

```
@@ -23,7 +23,7 @@
     "micro-cors": "0.1.1",
     "mobx": "6.7.0",
     "next": "12.3.4",
-    "next-connect": "0.9.1",
+    "next-connect": "^0.12.1",
```

Sugestão pra garantir que ele pegue a versão certa, além do `package-lock.json`

```suggestion
    "next-connect": "0.12.1",
```